cmd/samosa/main.go

```diff
@@ -3,11 +3,11 @@ package main
 import (
 	"log"
 
-	"github.com/burntcarrot/samosa/command"
+	"github.com/ehrktia/samosa"
```

Comment:
Could you please change it back to `burntcarrot`?

Reply:
@burntcarrot am not sure how to achieve this coz my PR is refactored of the functions used to report coverage, in case I point to `github.com/burntcarrot/samosa` it will fetch old functions. 
Let us decide on this after testing.

Reply:
Sure. Keeping this open for future reference.